canvas_modules/harness/test_resources/parameterDefs/readonlyTable_paramDef.json

```diff
@@ -0,0 +1,748 @@
+          "enum": [
+            "string",
+            "number",
+            "boolean",
+            "time",
+            "date"
+          ],
+          "default": ""
+        }
+      ]
+    }
+  ],
+  "uihints": {
+    "id": "readonlyTable.test",
+    "editor_size": "medium",
+    "label": {
+      "default": "ReadonlyTable Test"
+    },
+    "description": {
+      "default": "Test readonly table controls"
+    },
+    "parameter_info": [
+      {
+        "parameter_ref": "readonlyStructureTableControl",
+        "label": {
+          "default": "ReadonlyTable - structuretable"
+        },
+        "description": {
+          "default": "Readonly structuretable control with multi row selection",
+          "placement": "on_panel"
+        }
+      },
+      {
+        "parameter_ref": "readonlyStructurelistTableControl",
+        "label": {
+          "default": "ReadonlyTable - structurelisteditor"
+        },
+        "description": {
+          "default": "Readonly structurelisteditor with single row selection and a custom edit button label.",
+          "placement": "on_panel"
+        }
+      },
+      {
+        "parameter_ref": "readonlyNestedStructure",
+        "required": true,
+        "label": {
+          "default": "ReadonlyTable - nested Structurelisteditor"
+        },
+        "description": {
+          "default": "Readonly structuretable with a nested structurelisteditor.",
+          "placement": "on_panel"
+        }
+      },
+      {
+        "parameter_ref": "readonlyTableError",
+        "label": {
+          "default": "ReadonlyTable - error"
+        },
+        "description": {
+          "default": "readonlyTable control that has a non-valid field. Click the edit button to trigger the conditions.",
+          "placement": "on_panel"
+        }
+      },
+      {
+        "parameter_ref": "enableDisabledTable",
+        "label": {
+          "default": "Enable the disabled readonlyTable"
+        }
+      },
+      {
+        "parameter_ref": "readonlyTableDisabled",
+        "label": {
+          "default": "ReadonlyTable - disabled"
+        },
+        "description": {
+          "default": "readonlyTable that is disabled.",
+          "placement": "on_panel"
+        }
+      },
+      {
+        "parameter_ref": "showHiddenTable",
+        "label": {
+          "default": "Show the hidden readonlyTable"
+        }
+      },
+      {
+        "parameter_ref": "readonlyTableHidden",
+        "required": true,
+        "label": {
+          "default": "ReadonlyTable - hidden"
+        },
+        "description": {
+          "default": "readonlyTable control that was hidden because there are no rows in the table",
+          "placement": "on_panel"
+        }
+      }
+    ],
+    "complex_type_info": [
+      {
+        "complex_type_ref": "readonlyStructureTableControl",
+        "label": {
+          "default": "Rename Subpanel"
+        },
+        "control": "readonlyTable",
```

Comment:
I think the control would be under the parameter and not as part of the complex_type_info.  The schema doesn't support a `control` here.